tests/e2e-evm/test/abi_basic.test.ts

```diff
@@ -170,16 +170,29 @@ describe("ABI_BasicTests", function () {
           {
             name: "can be called by low level contract call",
             txParams: (ctx) => ({
-              to: caller.address,
+              to: lowLevelCaller.address,
```

Comment:
Execellent rename 👍. I was a bit lazy with the initial one -- lowLevelCaller gives better clarity on what this test is doing